CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 Below are the most important changes from each release.
 A more detailed list of changes is available in the corresponding milestones for each release in the Github issue tracker (https://github.com/googlefonts/fontbakery/milestones?state=closed).
 
+## 0.7.3 (2019-Apr-??)
```

Comment:
we are always targeting the Monday of the next week.

Reply:
OK, I amended that commit.